cpp/tools/clustertool.h

```diff
@@ -0,0 +1,24 @@
+#ifndef CERT_TRANS_TOOLS_CLUSTERTOOL_H_
+#define CERT_TRANS_TOOLS_CLUSTERTOOL_H_
+
+namespace cert_trans {
+
+
+template <class Logged>
```

Comment:
It doesn't *have* to be, it *could* totally be a `clustertool` which only works with instances of `LoggedCertificate` servers :)